model/host/hostutil.go

```diff
@@ -403,7 +403,13 @@ func (h *Host) ProvisioningUserData(settings *evergreen.Settings, creds *certdep
 				fetchCmd = []string{h.Distro.ShellBinary(), "-l", "-c", strings.Join(h.SpawnHostGetTaskDataCommand(), " ")}
 			}
 			var getTaskDataCmd string
-			getTaskDataCmd, err = h.buildLocalJasperClientRequest(settings.HostJasper, strings.Join([]string{jcli.ManagerCommand, jcli.CreateCommand}, " "), &options.Command{Commands: [][]string{fetchCmd}})
+			getTaskDataCmd, err = h.buildLocalJasperClientRequest(
+				settings.HostJasper,
+				strings.Join([]string{jcli.ManagerCommand, jcli.CreateCommand}, " "),
```

Comment:
Since you're changing the payload to use tags (and also making a request to a different gRPC method), the CLI command has to change from the `create-command` subcommand to the `create-process` subcommand.

Reply:
Oh i see, I misunderstood what you wanted changed.